src/Symfony/Component/Process/Tests/ProcessTest.php

```diff
@@ -48,6 +48,23 @@ protected function tearDown()
         }
     }
 
+    /**
+     * @group legacy
+     * @expectedDeprecation The provided cwd does not exist. Command is currently ran against getcwd(). This behaviour is deprecated since version 3.4 and will be removed in 4.0.
+     */
+    public function testInvalidCwd()
+    {
+        // Check that it works fine if the CWD exists
+        $cmd = new Process('echo test > testing1.txt', __DIR__);
```

Comment:
there is no need to pipe the output to a file

Reply:
Done